Dockerfile

```diff
@@ -4,7 +4,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     build-essential git gcc libc-dev libffi-dev bash make apt-utils
 WORKDIR /go/package-scanner/
 COPY . .
-RUN make tools
+
+ARG TARGETPLATFORM
+# TODO(tjonak): not sure whether I need to expose TARGETPLATFORM to make or is ARG automatically available
```

Comment:
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

Reply:
This comment was questioning whether I need to do `TARGETPLATFORM=$TARGETPLATFORM make tools` or can I just `make tools` and `TARGETPLATFORM` will be automatically in environmental variables visible from `make tools` since its an `ARG`.